NAMESPACE

```diff
@@ -68,6 +68,7 @@ importFrom(ggplot2,theme)
 importFrom(lubridate,ymd)
 importFrom(posterior,as_draws_list)
 importFrom(posterior,subset_draws)
+importFrom(rlang,abort)
```

Comment:
where is this used? I only see `cli::cli_abort()` calls (note that `cli::cli_abort()` calls `rlang::abort()` under the hood, but you shouldn't need to import the name.

Reply:
I can try removing it? I don't think it is used anywhere

Reply:
Oh wait this gets generated automatically, so it must be in here... let me look

Reply:
Ok nope this is what I get for copying things from other people without fully understanding them :) @zsusswein has set up his package functions to explicitly do the importfrom in the function, and had `@importFrom rlang abort` in his function that I modified